### ISSUE TITLE
Show upcoming weather data in agenda panel when Weather widget is toggled off

### ISSUE THREAD

**Issue by Amorphous404** [8bitPit/Niagara-Issues#2064]
### Is your feature request related to a problem?

Yes, i can't make the upcoming weather data appear when weather widget is turned off in agenda panel with (niagara://agenda) formula

### Describe the solution you'd like

Please make it possible to show weather data in agenda panel even when weather widget is turned off, just like the upcoming calender event, so that when users disable all the Niagara widget and Call the agenda panel in Kustom Apps with (niagara://agenda) formula the upcoming whether should appear how when weather widget is toggled on

https://user-images.githubusercontent.com/96917972/191723172-307a3761-453a-48d2-8386-2e46ef101681.mp4

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by Oddward** [comment]
It does make sense to support this tbh, but that would also mean Niagara would need location data even when the weather widget is turned off 🤔

**Comment by topsy-turvy99** [comment]
https://user-images.githubusercontent.com/96917972/193213732-5a966709-3219-4ba1-93d9-d1e06b1eb2b9.mp4

Well, according to me it makes sense to include the upcoming weather data in the agenda panel since Niagara has integrated support for kustom apps and offers a formula to call the panel

what doesn't make sense to me is that the niagara agenda can be called without niagara://calender formula, just by tapping on the invisible bar, that's always available even after removing all the items from favorites page (as you can see in the recording)

Now, even if a user wants to include the weather data, one has to enable the widget, which in turn spoils the setup and creates an overlay effect and cause delay when using swipe up for search

Not necessarily, location data can be turned off as usual if users turn off both the widget and upcoming forecasts, i thought it can kinda replicate the same behaviour similar to calendar events, that way it'll also help those users who want to make use of weather widget (todays forcecast) and don't want the other upcoming weather data to be included along with the calendar events

**Comment by Oddward** [comment]
Makes sense since some people only need to check the weather at certain times like before going out in the morning - a main use case for the agenda - but it would require new toggles to support that, and it's not a common request as far as I've seen. 



The invisible bar that shows the agenda was left unfixed because a lot of users appreciate having the ability to use it without affecting their setup and having a little bit of padding there (if it adds that) - from bug to feature. Admittedly, I think that was there since before the agenda shortcut was available to launch in other ways.

**Comment by TFackler** [comment]
We took a look at this and we see how it would be a useful addition. 

The downside we see though is that the group which would benefit from this is rather small (users who want to see the weather in the agenda but not on their home screen). Thus we have decided that we will add this functionality only if there are additional requests for this, so I'm closing the issue for now.

**Comment by shivangc22** [comment]
Would like to raise the same issue. In my opinion an ideal solution would be to copy the existing method i.e display the weather in the agenda screen only, thus accessing both by using the "niagara://agenda" url only.

**Comment by cnschn** [comment]
I have this exact use case as well, I'd like to use klwp to display information on my home screen, but have weather data in the agenda.
